readme.md

```diff
@@ -30,8 +30,8 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://<hatevadym>.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://<hatevadym>.github.io/layout_catalog/report/html_report/)
```

Comment:
```suggestion
- [DEMO LINK](https://hatevadym.github.io/layout_catalog/)
- [TEST REPORT LINK](https://hatevadym.github.io/layout_catalog/report/html_report/)
```

Reply:
Your demo links don't work. Please, fix them and provide into the PR description